food/export.py

```diff
@@ -207,12 +207,21 @@ def to_process(activity):
         IMPACTS_JSON, processes_corrected_impacts
     )
 
+    activities_to_export = []
+
+    # To be in sync with the ingredient editor, we need to remove the land_occupation for non ingredients
```

Comment:
I don't see any removed land_occupation in the `activities.json` diff, why ?

If we want to remove `land_occupation` for non ingredients shouldn't we don't compute land_occupation  in `compute_land_occupation()` ?

Reply:
Yes you’re right, this is something I tried but that was not working at the time… and now it works 🤷‍♂️ I did the change.

Regarding `activities.json` you're right about the diff, and maybe my statement about the fact that `land_occupation` was exported to `activities.json` was wrong and just the result of something else (like bad usage of the ingredient editor at the same time). Anyway I think that doing the check can't do any arm.